README.md

```diff
@@ -31,13 +34,16 @@ out:
   apikey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXX'
   list_id: 'XXXXXXXXXX'
   update_existing: false
+  double_optin: false
+  email_column: e mail
+  fname_column: first name
+  lname_column: last name
```

Comment:
If single quotes are not required, then no need to revert. Please rename 'e mail' to 'e-mail'. Thanks!